tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -359,6 +359,7 @@ connect_redirect_test(
     uint32_t bytes_received = 0;
     char* received_message = nullptr;
     uint64_t authentication_id;
+    std::string proxy_address_string = get_string_from_address(reinterpret_cast<const SOCKADDR*>(&proxy));
```

Comment:
Can "proxy" be null?

Reply:
No proxy will not be null in these tests.